addons/hearing/functions/fnc_updateHearingProtection.sqf

```diff
@@ -56,4 +61,18 @@ if (_goggles != "") then {
 
     private _attenuation = getNumber (_gogglesConfig >> QGVAR(lowerVolume)) min 1;
     GVAR(volumeAttenuation) = GVAR(volumeAttenuation) * (1 - _attenuation);
+
+    _hasBuiltInEHP = _hasBuiltInEHP || { getNumber (_gogglesConfig >> QGVAR(hasEHP)) > 0 };
+};
+
+// Unequip electronic hearing protection when gear has one built in
+if (_hasEHPOn && _hasBuiltInEHP) then {
+    [ACE_player, false] call FUNC(removeEHP);
```

Comment:
If a unit's inventory is full and currently has EHP (from inventory item) equipped, then proceeds to equip a helmet with EHP built in, it won't remove the EHP. Do we care about that?